packages/nuxt/src/module.ts

```diff
@@ -0,0 +1,48 @@
+import { defineNuxtModule, addServerHandler, addImportsDir, createResolver } from '@nuxt/kit';
+import { defu } from 'defu';
+import { type NuxtModule } from 'nuxt/schema';
+
+import { defaults } from './runtime/utils/constants';
+import { type LogtoRuntimeConfig, type LogtoRuntimeConfigInput } from './runtime/utils/types';
+
+// This will not export the default export
+export * from '@logto/node';
+export { default as LogtoNodeClient } from '@logto/node';
+
+const logtoModule: NuxtModule<LogtoRuntimeConfigInput> = defineNuxtModule<LogtoRuntimeConfigInput>({
+  meta: {
+    name: '@logto/nuxt',
+    configKey: 'logto',
+  },
+  defaults,
+  setup(options, nuxt) {
+    // Merge runtimeConfig with module options (options already merged with defaults)
+    const runtimeConfig = defu<LogtoRuntimeConfig, LogtoRuntimeConfigInput[]>(
+      nuxt.options.runtimeConfig.logto,
+      options,
+      {
+        fetchUserInfo: false,
```

Comment:
what does falsy `fetchUserInfo` mean? Can authenticated users request user info if this is false?

Reply:
see [this comment](https://github.com/logto-io/js/pull/639/files#diff-ec13d0fe2e97302e7f9d58953bd1c95b04f3efea529b3257bc8bf17300e5bb5bR17-R23)